### PR TITLE
Add initial support for ZWO AM mounts.

### PR DESCRIPTION
**Pull request by ddompe**
- Properly identifies the mounts. Tested on AM3 and AM5.
- Commands implemented but waiting for a clear day for testing

Posting here for initial comments or in case is useful for someone else.

### CODE REVIEW

**Comment by GreatAttractor** [comment]
Great, thanks for adding this! I'll have a look in the next few days.

You can test indoors by setting something in [this line](https://github.com/GreatAttractor/vidoxide/blob/9b83bf26ce8943be669680794b9d04520df70583/src/workers/capture.rs#L120) (e.g., `= Some([0.5, 0.0]`), putting a camera with a short/medium-f.l. lens on the mount and pointing it at an opposite wall of the room. Should be possible to set the drift speed close to angular speed of the mount tracking at 1x sidereal. Then, by changing the drift vector, you can simulate polar misalignment - and test guiding. (That's how I was testing the iOptron driver.)

**Comment by ddompe** [comment]
@GreatAttractor thanks for the tip, I will give it a try! I have a doubt: enabling tracking must use sidereal tracking rate by default? No support to specify solar or lunar tracking rate? I assume doesn't matter because it will be aligning anyway...

**Comment by GreatAttractor** [comment]
> enabling tracking must use sidereal tracking rate by default?

Yes, currently the "Enable sky tracking" button (which does `Mount::set_tracking`) is meant to use sidereal rate. We can add lunar/solar selection in the future.

File: src/mount/zwo.rs [hunk 1]
```diff
@@ -0,0 +1,283 @@
...
+            return Err("cannot guide when tracking is disabled".into());
+        }
+
+        const MIN_SIDEREAL_MULT: f64 = 0.1;
+
+        let a1_s = (axis1_speed.0.abs() / SIDEREAL_RATE.0).max(MIN_SIDEREAL_MULT);
+        let a2_s = (axis2_speed.0.abs() / SIDEREAL_RATE.0).max(MIN_SIDEREAL_MULT);
+
+        if axis1_speed.is_zero() {
+            send_cmd_and_get_reply(&mut self.serial_port, ":Mgn0000#".into(), ResponseType::None).map(|_| ())?;
```
Comment (GreatAttractor) [inline src/mount/zwo.rs]: It seems it should be `e`, `w` in this block (axis1 = RA) and `n`, `s` in the next block (axis2 = Dec),  similarly in blocks below.

Comment (ddompe) [reply src/mount/zwo.rs]: You are right, thanks

**Comment by ddompe** [comment]
Got clear skies last Saturday, was able to test it with the Ha scope. I adjusted the guide pulses for 500ms by default. Looks good enough to me for others to use.

**Comment by ddompe** [comment]
I also added some of the slewing rates typical of the ZWO mounts to the list of options available, otherwise it's basically <16x or Max.

**Pull request merged**